packages/e2e-tests/specs/widgets/editing-widgets.test.js

```diff
@@ -633,12 +633,14 @@ describe( 'Widgets screen', () => {
 
 		// Wait for the Legacy Widget block's preview iframe to load.
 		const frame = await new Promise( ( resolve ) => {
-			const checkFrame = async ( candidateFrame ) => {
-				const url = await candidateFrame.url();
-				if ( url.includes( 'legacy-widget-preview' ) ) {
+			const checkFrame = async () => {
```

Comment:
> @anton-vlasenko I believe you just have to remove the commits that's already on that branch and rebase your branch onto that.

Done.

Reply:
I've changed this because otherwise e2e tests fail.
We cannot "detect"  the needed frame by its URL because we've removed the `src` attribute.
So now we are selecting the related iframe element by its class name. Then we get the needed frame using the iframe element mentioned above.

Reply:
Oh cool, yeah!

We've experienced some flaky situations with iframe stuffs though so it'd be cool if we can re-run this several times before we merge this 👍.